tests/test_marc.py

```diff
@@ -0,0 +1,195 @@
+import transmogrifier.models as timdex
+from transmogrifier.helpers import parse_xml_records
+from transmogrifier.sources.marc import Marc
+
+
+def test_marc_record_all_fields_transform_correctly():
+    marc_xml_records = parse_xml_records(
+        "tests/fixtures/marc/marc_record_all_fields.xml", "record"
+    )
+    output_records = Marc("alma", marc_xml_records)
+    assert next(output_records) == timdex.TimdexRecord(
+        source="MIT Alma",
+        source_link=(
+            "https://mit.primo.exlibrisgroup.com/discovery/fulldisplay?vid="
+            "01MIT_INST:MIT&docid=alma990027185640106761"
+        ),
+        timdex_record_id="alma:990027185640106761",
+        title="Célébration : 10 siècles de musique de noël.",
+        alternate_titles=[
+            timdex.AlternateTitle(
+                value="Main Entry Uniform Title.", kind="Main Entry - Uniform Title"
+            ),
+            timdex.AlternateTitle(value="Uniform Title.", kind="Uniform Title"),
+            timdex.AlternateTitle(
+                value="Varying Form Of Title.", kind="Varying Form of Title"
+            ),
+            timdex.AlternateTitle(
+                value="Added Entry Uniform Title.",
+                kind="Added Entry - Uniform Title",
+            ),
+            timdex.AlternateTitle(
+                value="Added Entry Uncontrolled Related/Analytical Title.",
+                kind="Added Entry - Uncontrolled Related/Analytical Title",
+            ),
+        ],
+        call_numbers=["MA123.4", "123.45"],
+        citation=(
+            "Célébration : 10 siècles de musique de noël.. "
```

Comment:
MARC punctuation makes the citation a bit ugly, not sure if/how we want to address that